ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/ApplicationForm/ApplicationFormIntegrationTests.cs

```diff
@@ -1,29 +1,179 @@
-﻿using FluentAssertions;
+﻿using ApplyToBecome.Data.Models;
+using ApplyToBecome.Data.Models.Application;
+using FluentAssertions;
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
+using System.Linq.Dynamic.Core;
 using System.Text;
 using Xunit;
 
 namespace ApplyToBecomeInternal.Tests.Pages
 {
 	public class ApplicationFormIntegrationTests : BaseIntegrationTests
 	{
+		private AcademyConversionProject _project;
+		private Application _application;
+
 		public ApplicationFormIntegrationTests(IntegrationTestingWebApplicationFactory factory) : base(factory)
 		{
+			_project = AddGetProject();
+			_application = AddGetApplication(app => 
+				{
+					app.ApplicationId = _project.ApplicationReferenceNumber;
+				});
 		}
 
 		[Fact]
-		public async void The_project_template_link_is_present()
+		public async void The_application_form_link_is_present()
 		{
-			var projects = AddGetProjects().ToList();
-			var firstProject = AddGetProject(p => p.Id = projects.First().Id);
-
-			await OpenUrlAsync($"/school-application-form/{firstProject.Id}");			
+			await OpenUrlAsync($"/school-application-form/{_project.Id}");			
 
 			var pageItem = Document.QuerySelector($"#application-form-link");
 			pageItem.TextContent.Should().Be("Open school application form in a new tab");
-			pageItem.BaseUri.Should().Be($"http://localhost/school-application-form/{firstProject.Id}");
+			pageItem.BaseUri.Should().Be($"http://localhost/school-application-form/{_project.Id}");
```

Comment:
Is this going to be the correct url on the build server?

Reply:
Thanks. Changed to check just the end of the url